src/CacheCow.Server/CachingHandler.cs

```diff
@@ -1,646 +1,654 @@
-﻿using System;
```

Comment:
Thanks @IDisposable 
I might have to accept then change line endings or ask for a pull request with correct line endings